custom_config_data/config.yaml

```diff
@@ -61,6 +61,8 @@ INACTIVITY_SCORE_RECOVERY_RATE: 16
 EJECTION_BALANCE: 28000000000
 # 2**2 (= 4)
 MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
```

Comment:
```suggestion
# 2**16 (= 65,536)
CHURN_LIMIT_QUOTIENT: 65536
# [New in Deneb:EIP7514] 2**3 (= 8)
MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
```

Reply:
note, this simply is the default value from https://github.com/ethereum/consensus-specs/blob/v1.4.0-beta.6/configs/mainnet.yaml

so, technically it is correct even when omitted, as this config is based on `PRESET_BASE = 'mainnet'`